content/chainguard/libraries/python/build-configuration.md

```diff
@@ -238,6 +265,23 @@ Proceed to build your project:
 poetry build
 ```
 
+For [direct access](#direct-access) to Chainguard Libraries for Python with
+Poetry, use your username `CG_PULLTOKEN_USERNAME` and password
```

Comment:
```suggestion
Poetry, use your username and password
```

Reply:
I am leaving that in to be explicit for the example